decidim-elections/app/views/decidim/elections/admin/answers/index.html.erb

```diff
@@ -30,6 +30,10 @@
                 <%= answer.linked_resources(:proposals, "related_proposals").count %>
               </td>
               <td class="table-list__actions">
+                <% if question.total_votes.positive? %>
```

Comment:
I guess this is going to change when we have an election status from the BB, right?

Reply:
Yes, this is only for the time we mock the results. Later, we'll use the election status to handle if the results are visible or not.